gplv3-exceptions.txt

```diff
@@ -0,0 +1,4 @@
+As a special exception, the copyright holders of Errbot hereby grant permission 
+for plug-ins and script add-ons to be used with Errbot, provided that you also 
+meet the terms and conditions of the licenses of those plug-ins and script
+add-ons.
```

Comment:
To avoid any ambiguity, I think we should change the wording to:

> As a special exception, the copyright holders of Errbot hereby grant permission
> for plug-ins and script add-ons not bundled and distributed as part of errbot itself,
> potentially licensed under a license other than the GPL, to be used with Errbot,
> provided that you also meet the terms and conditions of the licenses of those
> plug-ins and script add-ons.

This emphasizes that it applies to third-party plugins rather than core plugins shipped as part of errbot itself and further emphasizes that it's OK to license said plugins under a license other than the GPL.